pandas/core/indexing.py

```diff
@@ -1167,16 +1164,14 @@ def _validate_read_indexer(
         if missing:
             if missing == len(indexer):
                 raise KeyError(
-                    "None of [{key}] are in the [{axis}]".format(
-                        key=key, axis=self.obj._get_axis_name(axis)
-                    )
+                    f"None of [{key}] are in the [{self.obj._get_axis_name(axis)}]"
```

Comment:
nitpick: can you define `axis_name = self.obj._get_axis_name(axis)` on the previous line and use `axis_name` inside the fstring?

Reply:
Done.